docs/README.md

```diff
@@ -1,22 +1,21 @@
 # Data Quality Committee Rules
 
-The rules linked below are either available for public review or have been approved by the DQC.
```

Comment:
This text should not be changed, as there is both approved and public exposure content in this branch/release.